yamlprivateh.go

```diff
@@ -122,46 +122,46 @@ func is_printable(b []byte, i int) bool {
 }
 
 // Check if the character at the specified position is NUL.
-func is_z(b []byte, i int) bool {
+func isZ(b []byte, i int) bool {
```

Comment:
Use isNullChar because here the z is not clear

Or isZeroChar

Reply:
This is out of scope.